.github/workflows/nmodl-doc.yml

```diff
@@ -58,46 +58,31 @@ jobs:
       - name: Install Python3 dependencies
         working-directory: ${{runner.workspace}}/nmodl
         run: |
-          pip3 install -U pip setuptools
-          pip3 install --user -r requirements.txt
-
-      # This step will set up an SSH connection on tmate.io for live debugging.
-      # To trigger it, simply add 'live-debug-docs' to your last pushed commit message.
-      - name: live debug session on failure
-        if: failure() && contains(github.event.head_commit.message, 'live-debug-docs')
-        uses: mxschmitt/action-tmate@v3
+          pip3 install -U pip setuptools 'pip-tools>=7.4.0'
+          pip3 install --user -r <(pip-compile --all-build-deps --all-extras --no-strip-extras 2>&1)
```

Comment:
Isn't installing these dependencies part of `generate_docs.sh`?

Reply:
Indeed.